android-kotlin/QuickStartTasks/app/src/main/java/live/ditto/quickstart/tasks/TasksApplication.kt

```diff
@@ -56,6 +56,9 @@ class TasksApplication : Application() {
         ditto.updateTransportConfig { config ->
             // Set the Ditto Websocket URL
             config.connect.websocketUrls.add(webSocketURL)
+
+            // Temporary workaround to prevent crash
+            config.peerToPeer.wifiAware.enabled = false
```

Comment:
We could leave this on for newer versions. 

```suggestion
            if (Build.VERSION.SDK_INT <= 27) {
                // Temporary workaround to prevent crash on Android 8.1 and below
                config.peerToPeer.wifiAware.enabled = false
            }
```

note: code not tested.